imap_processing/ultra/l1b/de.py

```diff
@@ -59,7 +58,7 @@ def calculate_de(de_dataset: xr.Dataset, name: str) -> xr.Dataset:
     energy = np.full(len(de_dataset["epoch"]), np.nan, dtype=np.float32)
     # TODO: uint8 fills with zeros instead of nans.
     #  Confirm with Ultra team what fill values and dtype we want.
-    species_bin = np.full(len(de_dataset["epoch"]), np.nan, dtype=np.uint8)
+    species_bin = np.full(len(de_dataset["epoch"]), "UNKNOWN", dtype="U10")
```

Comment:
If I understand it correctly, we want an integer type with a clear NaN value. Could we  use a slightly larger signed int type and set NaNs as -1 or -max_value?